pinot-core/src/test/java/org/apache/pinot/queries/ForwardIndexHandlerReloadQueriesTest.java

```diff
@@ -480,6 +490,47 @@ public void testSelectWithAggregationQueriesWithReload()
         }));
     List<Object[]> beforeResultRows3 = resultTable.getRows();
 
+    // TEST4
```

Comment:
I think since multiple functionalities are being tested here, now this file has become a bit hard to read and review imo.

I suggest adding brief comments / demarcate tests to indicate what is being tested where. Will also be more readable for future changes.